src/BenchmarksApps/TodosApi/TodosApi.csproj

```diff
@@ -9,12 +9,31 @@
     <LangVersion>preview</LangVersion>
     <UserSecretsId>b8ffb8d3-b768-460b-ac1f-ef267c954c85</UserSecretsId>
     <PublishAot>true</PublishAot>
-    <EnableLogging Condition="$(Configuration.StartsWith('Debug'))">true</EnableLogging>
+    <OpenApiDocumentsDirectory>.\</OpenApiDocumentsDirectory>
+    <!--<OpenApiEnabled>false</OpenApiEnabled>-->
+    <!-- Don't use configuration binding generator until bug is fixed: https://github.com/dotnet/runtime/issues/83600 -->
+    <EnableConfigurationBindingGenerator>false</EnableConfigurationBindingGenerator>
+    <EnableLogging Condition=" '$(EnableLogging)' == '' and $(Configuration.StartsWith('Debug')) ">true</EnableLogging>
     <DefineConstants Condition=" '$(EnableLogging)' == 'true' ">$(DefineConstants);ENABLE_LOGGING</DefineConstants>
   </PropertyGroup>
-
+  
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="8.0.0-preview.*" />
+    <ProjectReference Include="..\AspNetCore.OpenApi\AspNetCore.OpenApi.csproj" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="8.0.0-preview.4.*" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="8.0.0-preview.4.*" />
     <PackageReference Include="Npgsql" Version="8.0.0-preview.3" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.5.0" />
+    <PackageReference Include="Microsoft.Extensions.ApiDescription.Server" Version="8.0.0-preview.4.*">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
   </ItemGroup>
+
+  <ItemGroup>
+    <Content Update="appSettings.Development.json" CopyToPublishDirectory="false" />
+    <!-- Workaround for https://github.com/dotnet/aspnetcore/issues/47941 -->
+    <IlcArg Include="--nopreinitstatics" />
```

Comment:
Can we delete this now that https://github.com/dotnet/runtime/pull/85506 is fixed?

Reply:
Has that fixed flowed to installers yet?

Reply:
I don't believe so, but we don't need it to. The benchmarks use the latest daily build of runtime.

Reply:
It has not (talking about `main` here). While this would run on the perf infrastructure 'cos it pulls the latest individual runtime installers, it would be very difficult to run this on dev machines until the fix flows to the `main` SDK installer, so I think we shouldn't be removing workarounds until that happens.

Reply:
It's coming in https://github.com/dotnet/installer/pull/16275. Hasn't been merged yet.